CONTRIBUTING.md

```diff
@@ -81,7 +81,17 @@ The following requirements must be fulfilled for a Pull request to be mergable t
 * If the Pull request requires changes in structure of user files (e.g. project/synth or other xml files) or flash configuration.
     * It must ensure that files or flash configuration created with the official firmware or previous community releases stay compatible or are automatically upgraded (upward compatibility).
     * If possible files and flash configuration created with the changes of the Pull request can be used with older firmwares not having the change (downward compatibility). Older firmwares must not break if a newer configuration was stored on the device.
-* If the Pull request changes end user behavoir or introduces new features a new entry in the [CommunityFeatures.md](CommunityFeatures.md) file needs to be created in the preexisting style describing the feature and it's options as a small manual to users. This includes all runtime and compile time flags which shall be named in respective sections.
+* If the Pull request changes end user behavoir or introduces new features a new entry in the [CommunityFeatures.md](Documentation/CommunityFeatures.md) file needs to be created in the preexisting style describing the feature and it's options as a small manual to users. This includes all runtime and compile time flags which shall be named in respective sections.
+
+### UI Changes
+
+Changes to the existing UI should follow the following process
+
+* Ensure they meet the guidelines set out in [UX Principles](Documentation/ux_principles.md)
+* Place the change in the community feature menu for one beta cycle
+* Following a full beta cycle and user feedback, open a poll on the feature
+* Given maintainer approval and positive poll results, the feature can be moved from the community menu to the default Ui
```

Comment:
I feel like this is not explicit enough. What is a positive poll result, more than 50%, 100%? I did originally intend all settings that change existing behavior to stay in community settings and everything else integrated in the main menu. If we want to make community settings new standard behavior I agree we need a mechanism. Suggestion: At the end of every beta cycle there is a community meeting on discord where people can put up stuff to vote on and 90% of the participants must agree. (I'm flexible on the 90%, I'm sure there is prior art for voting systems)

Reply:
Yeah I left it somewhat open since I don't think we'll be able to prescribe a complete process until we actually try it out. I like the idea of the Discord community meeting though.